.circleci/config.yml

```diff
@@ -0,0 +1,169 @@
+      - git_fetch_all_tags
+      - install_node
+      - generate_random_wallet
+      - when:
+          condition: <<parameters.linux>>
+          steps:
+            - run:
+                name: Check Go Version
+                command: |
+                  v=`go version | { read _ _ v _; echo ${v#go}; }`
+                  if [[ $v != `cat GO_VERSION_MIN` ]]; then
+                    echo "GO_VERSION_MIN file does not match the go version being used."
+                    echo "Please update image to cimg/go:`cat GO_VERSION_MIN` or update GO_VERSION_MIN to $v."
+                    exit 1
+                  fi
+            - run: sudo apt-get update
+            - run: sudo apt-get install ocl-icd-opencl-dev libhwloc-dev
+            - run: sudo apt-get install python-is-python3
+
+      - when:
+          condition: <<parameters.darwin>>
+          steps:
+            - run:
+                name: Install Go
+                command: |
+                  curl https://dl.google.com/go/go`cat GO_VERSION_MIN`.darwin-<<parameters.darwin-architecture>>.pkg -o /tmp/go.pkg && \
+                  sudo installer -pkg /tmp/go.pkg -target /
+            - run:
+                name: Export Go
+                command: |
+                  echo 'export GOPATH="${HOME}/go"' >> $BASH_ENV
+            - run: go version
+            - run:
+                name: Install dependencies with Homebrew
+                command: HOMEBREW_NO_AUTO_UPDATE=1 brew install pkg-config coreutils jq hwloc
+            - run:
+                name: Install Rust
+                command: |
+                  curl https://sh.rustup.rs -sSf | sh -s -- -y
+      - run: git submodule sync
+      - run: git submodule update --init
+  download-params:
+    steps:
+      - restore_cache:
+          name: Restore parameters cache
+          keys:
+            - 'v26-2k-lotus-params'
+          paths:
+            - /var/tmp/filecoin-proof-parameters/
+      - run:  ./lotus fetch-params 2048
+      - save_cache:
+          name: Save parameters cache
+          key: 'v26-2k-lotus-params'
+          paths:
+            - /var/tmp/filecoin-proof-parameters/
+  install_ipfs:
+    steps:
+      - run: |
+          curl -O https://dist.ipfs.tech/kubo/v0.16.0/kubo_v0.16.0_linux-amd64.tar.gz
+          tar -xvzf kubo_v0.16.0_linux-amd64.tar.gz
+          pushd kubo
+          sudo bash install.sh
+          popd
+          rm -rf kubo
+          rm kubo_v0.16.0_linux-amd64.tar.gz
+  git_fetch_all_tags:
+    steps:
+      - run:
+          name: fetch all tags
+          command: |
+            git fetch --all
+  install_node:
+    steps:
+      - run: curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
+      - run: export NVM_DIR="$HOME/.nvm"
+      - run: echo 'export NVM_DIR=$HOME/.nvm' >> $BASH_ENV
+      - run: echo 'source $NVM_DIR/nvm.sh' >> $BASH_ENV
+      - run: nvm install v14
+      - run: nvm use v14
+      - run: nvm alias default v14
```

Comment:
Should probably use v19?

Reply:
I tested v19, and it doesn't work. It seems that Polyphene deveoped it with an older version. Updating to v19 can be a future task.